scripts/protein/esm2/test_esm2_pretrain.py

```diff
@@ -62,7 +61,6 @@ def dummy_protein_dataset(tmp_path):
     return db_file
 
 
-@pytest.mark.skip("duplicate unittest")
 @pytest.fixture
 def dummy_parquet_train_val_inputs(tmp_path):
```

Comment:
now that you're at it you may want to consider importing these fixtures from bionemo.testing